tools/bin/mbedtls-rewrite-branch-style

```diff
@@ -385,6 +392,33 @@ class CodeStyleBranchRewriter(MbedTLSBranchRewriter):
             ok = False
         return ok
 
+    def maybe_fetch(self, ref: str) -> None:
+        """Fetch the given ref. Do not update any local branches.
+
+        If the ref is of the form REMOTENAME/..., fetch a branch from that
+        remote. Otherwise, fetch from the remote configured with the
+        fetch_from parameter when instantiating this class.
+        """
+        # If ref is an acceptable git ref of any type, accept it.
+        # This means we won't try to update an existing remote ref which
+        # might have changed upstream since the last fetch.
+        if self.check_git(['rev-parse', '--quiet', '--verify', ref],
+                          stdout=subprocess.DEVNULL):
+            return
+        fetch_from = self.fetch_from
+        local = ''
+        pos = ref.find('/')
+        if pos != -1 and ref[:pos] in self.get_git_lines(['remote']):
+            fetch_from = ref[:pos]
+            local = ref
+            ref = ref[pos + 1:]
+        if fetch_from is None:
+            return
```

Comment:
Same here: I would suggest adding a self.info. This might be an error in command line parameters